electron/anticheat/titles_map.ts

```diff
@@ -0,0 +1,27 @@
+// this hash contains a mapping of titles as displayed in Epic to titles as
+// displayed in areweanticheatyet.com https://github.com/Starz0r/AreWeAntiCheatYet/blob/master/games.json
+//
+// comparison is case insensitive, so no need to add cases like 'VALORANT': 'Valorant'
+// if the epic title is contained, so no need to add cases like 'Fall Guts': 'Fall Guys: Ultimate Knockout'
+//
+// left is the epic title, right is the areweanticheatyet title
+const TITLES_MAP: { [key: string]: string } = {
+  'Battlefield™ 2042': 'Battlefield 2042',
+  'Back 4 Blood: Standard Edition': 'Back 4 Blood',
+  'Rainbow Six Siege': 'Rainbow Six: Siege',
+  'Phantasy Star Online 2 New Genesis': 'Phantasy Star Online 2',
+  'Knockout City™': 'Knockout City',
+  'Heroes & Generals WWII': 'Heroes & Generals',
+  'Crysis Remastered': 'Crysis',
+  'STAR WARS™ Battlefront™ Ultimate Edition': 'Star Wars: Battlefront (2015)',
+  'STAR WARS™ Battlefront™ II: Celebration Edition':
+    'Star Wars: Battlefront II (2017)',
+  'The Division': "Tom Clancy's The Division", // added this to prevent false match with The Division 2
+  'Battlefield V Definitive Edition': 'Battlefield V'
```

Comment:
But it would be great if areweanticheatyet could provide store ids for the games listed there. Like:

```jsonc
{
        "url": "",
        "name": "Battlefield 2042",
        "logo": "",
        "native": false,
        "status": "Broken",
        "reference": "https://www.protondb.com/app/1517290#atmpx3XrXL",
        "anticheats": [
            "Easy Anti-Cheat"
        ],
        "notes": [],
        "updates": []
        "ids": {
          "steam": "1517290",
          "epic": "<how-do-I-find-the-epic-game-id?>"
        }
},
```

Reply:
I'm ok with that, if they are ok with that it would make the check easier, but it's up to them (I understand it's out of the scope of areweanticheatyet and it would be another thing to maintain for them that they don't really use).

I don't know how to get Epic's id for arbitrary games, but I know that we can get information about a game in the metadata legendary downloads (`~/.config/Legendary/metadata/....json`). In that file we have the appName (what we use internatlly everywhere in heroic) and there's an `id` also in the metadata property but we don't really use it and I don't know if it is a game id (I don't know why Legendary uses the appName for everything either).

I also tought about something like `name_in_stores` instead of `ids` so it's easier to find out. But again, I think it might be outside of the scope of that project.

Reply:
> But again, I think it might be outside of the scope of that project.

Well, it's not as if they had to add this information. It's on best-effort basis. If they don't add it, we can add it.

Also, something to consider is that the map you created in this repository would only be evaluated on updates of Heroic. This could get out of sync with areweanticheatyet and would not allow to automatically discover the maps as we auto discover the areweanticheatyet.

Reply:
Yes, I was aware of the dependency of new heroic updates neede to update the map, but for now it's the best we can do I guess, or at least it's good enough for now, there's always room for improvements.

My main reason to add this feature was actually just fortnite, since we get A LOT of questions of users trying to install fortnite on linux.

Reply:
Hey, maintainer of AWACY here, I added the storeIds property to the list.  
Personally I'd like to store the `ID` (or the `appName` because you said it's used internally anyway) along with the `name_in_store` so that we can benefit from this as well ^^.

I'd also offer you that I can add all the missing `name_in_store` properties for Pull-Requests that want to add just the id.

Reply:
Since AWACY now supports `storeIds.epic.namespace`, we should remove this file `titles_map.ts` and add the `namespace` id to any mismatch to AWACY instead.

`namespace` ids for games can be easily found on https://database.egdata.app/offers/

Reply:
@felipecrs are you planning to add the namespace for the games in AWACY? I know it now supports the feature but I would like to have this feature merged and I don't know if I should wait until that's ready to be used or if I can merge this for now and then a refactor can be done using namespace when that's added to all games

Reply:
- https://github.com/Starz0r/AreWeAntiCheatYet/pull/564#issue-1299592641

Reply:
thanks!!